### DISCUSSION TITLE
Radio suggestion

### DISCUSSION THREAD

**Discussion by periode** [mikepierce/internet-radio-streams#7] [Station Suggestions]
Dial radio, an automated broadcasting people's playlists.

https://dialradio.live

:)

**Comment by mikepierce** [comment]
Done. 1c0aaed. This is a cool project :) 

Quick note: the `PUBLIC_MP3_SOURCE` URL listed in the homepage source HTML, [https://cast.dialradio.live/stream.mp3](https://cast.dialradio.live/stream.mp3), returns a 404 error.

**Reply by periode** [reply]
Thanks!

Huh, interesting that the MP3 url is still there... We have `aac` and `opus` streams [here](https://cast.dialradio.live)

Do you need mp3 specifically?

**Reply by mikepierce** [reply]
Nah I don't need mp3 specifically, thank you :) I've got the `aac` and `opus` stream links.

Quick note: the "M3U" links on the [Icecast2 Status page](https://cast.dialradio.live) point to files containing strange broken links.
